test/paymentmethods.collective.to.collective.test.js

```diff
@@ -0,0 +1,438 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import * as utils from './utils';
+import models from '../server/models';
+import * as store from './features/support/stores';
+
+const ORDER_TOTAL_AMOUNT = 1000;
+
+const createOrderQuery = `
+  mutation createOrder($order: OrderInputType!) {
+    createOrder(order: $order) {
+      id
+      fromCollective {
+        id
+        slug
+      }
+      collective {
+        id
+        slug
+      }
+      subscription {
+        id
+        amount
+        interval
+        isActive
+        stripeSubscriptionId
+      }
+      totalAmount
+      currency
+      description
+    }
+  }
+`;
+
+
+describe("payments.collectiveToCollective.test.js", () => {
```

Comment:
yeah I was really confused because today there is no standard so i tried to follow at least the same standard as the other paymentMethods test files( example: `paymentmethods.paypal.payment.test.js`). anyways, i'll change here to camelCase at least in this case for now and will create an issue to standardize files test to camelCase. thanks.